rest/model/host.go

```diff
@@ -12,6 +12,7 @@ import (
 type APIHost struct {
 	Id               *string    `json:"host_id"`
 	HostURL          *string    `json:"host_url"`
+	Tag              string     `json:"tag"`
```

Comment:
You'll need to add the fields to the BuildFromService/ToService functions below otherwise it won't be populated. This should be easier once we can code gen the structs!